src/services/authService.js

```diff
@@ -255,4 +255,37 @@ export default class AuthService {
 
     return 'User Role Assigned Successfully';
   }
+
+  /**
+   * @param {object} req reqest object
+   * @return {string} - success message;
+   */
+  static async rememberUserProfile(req) {
```

Comment:
You can apply restructuring here like this  `static rememberUserProfile( { body: { remember profile }, user: { I'd } })`

Reply:
ok

Reply:
OK. Noted